app/experimenter/experiments/api/v3/serializers.py

```diff
@@ -11,6 +11,7 @@ class ExperimentRapidSerializer(ChangelogSerializerMixin, serializers.ModelSeria
     rapid_type = serializers.HiddenField(default=Experiment.RAPID_AA_CFR)
     owner = serializers.ReadOnlyField(source="owner.email")
     slug = serializers.ReadOnlyField()
+    public_description = serializers.ReadOnlyField()
```

Comment:
Actually I think this can be a hidden field since we only wanna set it but we don't need it in the frontend....

Reply:
kk will do